server/routes/api/attachments/attachments.ts

```diff
@@ -291,15 +291,26 @@ const handleAttachmentsRedirect = async (
     }
   );
 
-  if (attachment.isPrivate) {
-    ctx.set(
-      "Cache-Control",
-      `max-age=${BaseStorage.defaultSignedUrlExpires}, immutable`
-    );
-    ctx.redirect(await attachment.signedUrl);
+  const { url, maxAge } = attachment.isPrivate
```

Comment:
I don't understand why these changes are needed at all still, I updated the component to just pass the attachment URL into the iframe directly and it worked fine… there should be no backend changes needed.